### ISSUE TITLE
Prelude does not export BlobSize

### ISSUE THREAD

**Issue by Sytten** [SeaQL/sea-orm#1543]
## Description

I updated some code from 0.10.3 to 0.11.1. The generated entities now have a column type with `BlobSize` set, but the prelude doesn't contain that export so it causes a compilation error. 

## Steps to Reproduce

1. Create Postgres table with a blob
2. Run the CLI to generate entities

### Expected Behavior

Prelude should export the `BlobSize` or the CLI should generate the proper import for it.

### Actual Behavior

`BlobSize` is missing

### Reproduces How Often

Always

### Workarounds

You import it manually, but it's not ideal since it gets overridden on the next generation 

## Versions

```
├── sea-orm v0.11.1
│   ├── sea-orm-macros v0.11.1 (proc-macro)
│   ├── sea-query v0.28.3
│   │   ├── sea-query-derive v0.3.0 (proc-macro)
│   ├── sea-query-binder v0.3.0
│   │   ├── sea-query v0.28.3 (*)
│   ├── sea-strum v0.23.0
│   │   └── sea-strum_macros v0.23.0 (proc-macro)
├── sea-query v0.28.3 (*)
├── sea-orm v0.11.1 (*)
```

**Comment by billy1624** [comment]
Hey @Sytten, welcome back! Just open a PR to address this issue

**Comment by Sytten** [comment]
Thanks! Love you guys so much. Didnt have time to contribute much (probably not much until april), but getting back into it.

**Comment by billy1624** [comment]
Has been fixed on [0.11.2](https://github.com/SeaQL/sea-orm/releases/tag/0.11.2)
